packages/@react-spectrum/searchwithin/stories/SearchWithin.stories.tsx

```diff
@@ -119,7 +119,9 @@ CustomWidth30.storyName = 'Custom width: 30';
 export const LabelPositionSide = () => render({labelPosition: 'side'});
 LabelPositionSide.storyName = 'labelPosition: side';
 
-export const NoLabel = () => render({label: undefined, 'aria-label': 'Aria Label'});
+export const NoVisibleLabel = () => render({label: undefined, 'aria-label': 'blahhaw'});
```

Comment:
Ah lol, sorry bout this scuffed aria-label
```suggestion
export const NoVisibleLabel = () => render({label: undefined, 'aria-label': 'Test aria label'});
```